gradle/libs.versions.toml

```
@@ -198,6 +200,7 @@ sentry = { module = "io.sentry:sentry", version.ref = "sentry-android" }
 
 zxing-core = { module = "com.google.zxing:core", version = "zxing-core" }
 zxing-android-embedded = { module = "com.journeyapps:zxing-android-embedded", version.ref = "zxing-android-embedded" }
+firebase-config-ktx = { group = "com.google.firebase", name = "firebase-config-ktx", version.ref = "firebaseConfigKtx" }
```

```suggestion
firebase-config-ktx = { group = "com.google.firebase", name = "firebase-config", version.ref = "firebaseConfigKtx" }
```
이거 ktx는 예전 라이브러리일거입니다. 지금은 `firebase-config`로 다 대체되었어.

참고 - https://firebase.google.com/support/release-notes/android

---

아 그리고 이거 라이브러리 없애야할듯? 사용하는 곳 없는거 보니

---

맞습니다 제거할게요!